### PR TITLE
docs: add riscv64 to install script

### PR DESCRIPTION
**Pull request by remkade**
I'm hoping to add riscv64 support to `dprint`, but feel free to reject if you aren't interested in another platform. This is a dev dependency for some important other projects that I'd love to add support for as well. In my experience riscv64 tends to be pretty well supported in Rust so I wouldn't expect any weird platform specific issues.

Adding risc64 to build matrix and install scripts. This adds the ability to use riscv64 plugins as well whenever those are built.

* Updating ci.generate.ts to include riscv64
* Generated new ci.yml
* Adding support for finding and loading riscv64 plugins
* Install script now includes support for riscv64

### CODE REVIEW

**Comment by Eveeifyeve** [comment]
There was a pr long time ago made at #830, I don't know what has happened since then.

**Comment by dsherret** [comment]
IIRC, it was blocked on the Wasmer upgrade which has now landed. I landed this for gnu in the original PR and updated this PR to just have the install script change. There's no support for musl atm from cross.

File: crates/dprint/src/plugins/implementations/process/setup_process_plugin.rs [hunk 1]
```diff
@@ -216,13 +216,13 @@ fn get_os_path<'a>(plugin_file: &'a ProcessPluginFile, environment: &impl Enviro
     "linux" => match arch.as_str() {
       "x86_64" => plugin_file.linux_x86_64.as_ref(),
       "aarch64" => plugin_file.linux_aarch64.as_ref().or(plugin_file.linux_x86_64.as_ref()),
-      "riscv64" => plugin_file.linux_riscv64.as_ref(),
+      "riscv64" => plugin_file.linux_riscv64.as_ref().or(plugin_file.linux_x86_64.as_ref()),
```
[the remark this answers is not in the document]
Comment (dsherret) [reply crates/dprint/src/plugins/implementations/process/setup_process_plugin.rs]: Reverted in order to land this PR, but feel free to resubmit it if I'm wrong.

**Review by dsherret — APPROVED** [review]
Thanks!

**Comment by dsherret** [comment]
I added a https://github.com/dprint/dprint-plugin-exec/pull/39 build for dprint-plugin-exec. Wasm plugins should work fine, but some other process plugins like dprint-plugin-prettier still won't work.

**Pull request merged**